dev-requirements.txt

```diff
@@ -1,4 +1,3 @@
 beautifulsoup4==4.9.3
-nose-randomly==1.2.6
```

Comment:
> Also nosetests have been removed as the tests have been migrated to pytests and they have been moved in order to show test coverage.

Would be good to get this context in the commit message so that people don't have to visit GitHub to find out the reason, it's all available locally :)

Reply:
Will add something to commit message